cloud/resource_pulsar_cluster.go

```diff
@@ -710,6 +711,23 @@ func resourcePulsarClusterDelete(ctx context.Context, d *schema.ResourceData, me
 	if err != nil {
 		return diag.FromErr(fmt.Errorf("ERROR_DELETE_PULSAR_CLUSTER: %w", err))
 	}
+	err = retry.RetryContext(ctx, 20*time.Minute, func() *retry.RetryError {
```

Comment:
Can use `schema.TimeoutDelete` to replace this hard-coded 20m

Reply:
fixed